frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -20,9 +40,52 @@ const Schedule = () => {
   const defaultEnd = new Date(defaultStart.getTime() + 28699999);
 
   const [curStart, setCurStart] = useState(defaultStart);
-  const [events, setEvents] = useState(tempEvents);
+  const [events, setEvents] = useState<CalEvent[]>([]);
+  const [calDrivers, setCalDrivers] = useState<CalendarDriver[]>([]);
+  const [drivers, setDrivers] = useState<Driver[]>([]);
   const [viewState, setviewState] = useState(false);
 
+  useEffect(() => {
+    fetch('/rides')
+      .then((res) => res.json())
+      .then(({ data }) => {
+        setEvents(data
+          .filter((ride: Ride) => ride.type !== "unscheduled" && ride.driver !== undefined && ride.startLocation !== undefined)
+          .map((ride: Ride) => ({
+            id: ride.id,
+            title:
```

Comment:
according to the designs, the title doesn't seem to need the rider and driver names. if you need them for the alert then you could just store them with the event and create the string in onSelectEvent. i think that'll also help simplify onEventDrop.

Reply:
yeah it makes sense to remove the driver's name since we already know that from the schedule header (which can simplify onEventDrop), but I'm thinking it might be better to just put the riders' names on there because if we want to move a ride under another driver, it might to know which ride is the one we want to move by looking at the riders' names (especially if there's a lot of rides at the same time?)

Reply:
I can go ask the designers about that